src/ovs/aca_ovs_control.cpp

```diff
@@ -264,6 +267,14 @@ void ACA_OVS_Control::parse_packet(uint32_t in_port, void *packet)
         aca_dhcp_server::ACA_Dhcp_Server::get_instance().dhcps_recv(
                 in_port, const_cast<unsigned char *>(payload));
       }
+
+      /* oam message procedure */
+      if (aca_oam_port_manager::Aca_Oam_Port_Manager::get_instance().is_oam_server_port(
```

Comment:
is_oam_server_port currently takes in a port ID and not udp_dport.

I think we can repurpose it to take in udp_dport and then go through a list of all Zeta enabled ports to see if it is one of the udp_dport for an OAM setup.